_posts/blog/2019-08-01-lab-meeting.md

```diff
@@ -0,0 +1,58 @@
+# Whitaker Lab Meeting | 1 August 2019
+
+This week’s lab meet was petite but by no means picayune! 
+We discussed ways to learn physics, Neurohackademy, and how to build trust and confidence in a group. 
+Thanks Kirstie, Isla, and Sarah for your tips, suggestions, and a super interesting conversation!
+
+### Celebrations and cool things to share:
+
+Kirstie: is finding Neurohackademy great so far – with lots of awesome talks! Here are a select few :point_down:
+  * Codeclimate for readable code: https://codeclimate.com
+  * Intro to git: http://rogerdudler.github.io/git-guide
+  * The most fantastic git resource: https://ohshitgit.com
+  * A bunch of thing you can do with Jupyer: https://blog.jupyter.org/i-python-you-r-we-julia-baf064ca1fb6
+  * A great project template for a research project: https://drivendata.github.io/cookiecutter-data-science
+
+Georgia (and Kirstie) got ethics approval for the Citizen Science Project – the day has finally arrived! YAY! 
+The GitHub repository and survey are now live, and we’ve sent an email to the project’s mailing list to launch it. :rocket:
+
+### Questions we’re thinking about: 
+
+Isla: is in the midst of writing her dissertation, and is wondering how to pinpoint the right level of detail needed.
+Because she’s used to sharing knowledge and making things clear :green_heart:, it can be heard to work out the right level     of explanation for other experts. 
+  * Tip for writing - start with bullet points and then build up a path of what else needs explaining.
+
+Georgia: wants to know how to help people help each other in the lab meetings. 
+  * Sarah suggested that since she doesn’t have the domain knowledge to help everyone, if we talked about technical challenges, she would be better able to help.
+  * Georgia suggested having some kind of shared goal which we can all work together on. 
+  * Isla suggested being explicit and bringing an extremely practical problem everyone could on.
+  * Sarah  suggested something like a coding club - maybe there could be a small problem but one entirely contained within the meeting? Something like REG tech talks. 
+
+Sarah: Wants tips on working on a project if the content is overwhelming (a.k.a anyone familiar with Guassian Processes? :sweat_smile:)
+
+Kirstie has been thinking a lot about the need for trust in communities – specifically, that we need to trust each other to be able to ask each other questions. 
+She’s been thinking about what the barriers to trust are in the group are. She’d love to know what has worked in the past to build a supportive team and environment.
+  * Georgia: Overall, it feels like there is a lot of trust at the Turing - people do help each other out.
```

Comment:
Can you reformat this into a summary of the discussion rather than individual points?

Fine to cut down on details - it doesn't need to be a transcript.

Happy to iterate if you're not sure 😄